src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+    private int randomNumberColor = random.nextInt(Color.values().length);
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[randomNumberColor].toString();
```

Comment:
as described in common mistakes, let's use name() method to get string representation of enums